core/src/lib.rs

```diff
@@ -15,10 +15,11 @@ pub use imgui;
 
 #[doc(hidden)]
 #[inline(always)]
-pub unsafe fn __init(arcdps: raw_structs::HANDLE) {
+pub unsafe fn __init(arcdps: raw_structs::HANDLE, name: &'static str) {
     __set_handle(arcdps);
     #[cfg(feature = "log")]
-    let _ = log::set_logger(&logging::LOGGER).map(|()| log::set_max_level(log::LevelFilter::Trace));
+    let _ = log::set_boxed_logger(Box::new(logging::ArcdpsLogger::new(name)))
+        .map(|()| log::set_max_level(log::LevelFilter::Trace));
```

Comment:
This would trigger a warning for consumers when they don't use the log feature. Could you change that so that this doesn't happen?

Reply:
Done